packages/fxa-content-server/app/scripts/templates/post_verify/password/force_password_change.mustache

```diff
@@ -0,0 +1,46 @@
+<div id="main-content" class="card force-password-change">
+    <header>
+        <h1 id="fxa-force-password-change-header">
+            {{#serviceName}}
+                <!-- L10N: For languages structured like English, the second phrase can read "to continue to %(serviceName)s" -->
+                {{#t}}Please change your password{{/t}} <span class="service">{{#t}}Continue to %(serviceName)s{{/t}}</span>
+            {{/serviceName}}
+            {{^serviceName}}
+                {{#t}}Please change your password{{/t}}
+            {{/serviceName}}
+        </h1>
+    </header>
+
+    <section>
+        <form novalidate>
+            <!-- hidden email field is to allow Fx password manager
+                 to correctly save the updated password. Without it,
+                 the password manager tries to save the old password
+                 as the username. -->
+            <input type="email" value="{{email}}" class="hidden" />
+
+            <p class="verification-email-message">
+                {{#t}}We detected suspicious behavior on your Firefox account. To prevent further unauthorized access to your Firefox Account, please change your password. This will sign you out of your other accounts everywhere it is being used.{{/t}}
```

Comment:
"...other _accounts_ ..." here could be confusing.

Reply:
suggestion:

> We detected suspicious activity on your Firefox Account and have signed you out of all devices for your protection. To prevent further unauthorized access please change your password.